lib/mapping/CDrawRoadsOperation.cpp

```diff
@@ -140,45 +140,47 @@ static bool ruleIsSomething(const std::string & rule)
 	return rule == "+";
 }
 
+/*
 static bool ruleIsAny(const std::string & rule)
 {
 	return rule == "?";
 }
+*/
```

Comment:
It is used in assert unless you build without them as release. Also I think formatting fixes in related code are reasonable, but please at least don't include unrelated files into commits. That RMG-related file clearly has nothing to do with adventure map spells.